tests/test_table.py

```diff
@@ -2233,15 +2233,17 @@ def bin_f(id_, x):
 
     def test_collapse_observations_by_metadata_one_to_many(self):
         """Collapse observations by arbitary metadata"""
-        dt_rich = Table(np.array([[5, 6, 7], [8, 9, 10], [11, 12, 13]]),
-                        ['1', '2', '3'], ['a', 'b', 'c'],
+        dt_rich = Table(np.array([[5, 6, 7], [8, 9, 10], [11, 12, 13],
```

Comment:
Instead of modifying the existing case, can you leave this one and add a new one to reproduce the bug?